features/support/response_support_methods.rb

```diff
@@ -23,6 +23,30 @@
     "combined_assessed_capital" => "result_summary.capital.combined_assessed_capital",
     "remarks" => "assessment.remarks",
   },
+  "v7" => {
+    "assessment_result" => "result_summary.overall_result.result",
+    "capital contribution" => "result_summary.overall_result.capital_contribution",
+    "income contribution" => "result_summary.overall_result.income_contribution",
+    "disposable_income_summary" => "result_summary.disposable_income",
+    "total outgoings and allowances" => "result_summary.disposable_income.combined_total_outgoings_and_allowances",
+    "partner allowance" => "result_summary.disposable_income.partner_allowance",
+    "dependant allowance" => "result_summary.disposable_income.dependant_allowance",
+    "capital summary" => "result_summary.capital",
+    "partner capital summary" => "result_summary.partner_capital",
+    "capital_lower_threshold" => "result_summary.capital.proceeding_types.0.lower_threshold",
+    "gross_income_upper_threshold_0" => "result_summary.gross_income.proceeding_types.0.upper_threshold",
+    "gross_income_upper_threshold_1" => "result_summary.gross_income.proceeding_types.1.upper_threshold",
+    "gross_income_proceeding_types" => "result_summary.gross_income.proceeding_types",
+    "main property" => "assessment.capital.capital_items.properties.main_home",
+    "additional property" => "assessment.capital.capital_items.properties.additional_properties.0",
+    "vehicle" => "assessment.capital.capital_items.vehicles.0",
+    "partner property" => "assessment.partner_capital.capital_items.properties.additional_properties.0",
+    "overall_disposable_income" => "result_summary.partner_disposable_income",
+    "employment" => "result_summary.disposable_income.employment_income",
+    "partner_employment" => "result_summary.partner_disposable_income.employment_income",
+    "combined_assessed_capital" => "result_summary.capital.combined_assessed_capital",
+    "remarks" => "assessment.remarks",
+  },
```

Comment:
This seems unnecessary - v6 and v7 have the same response schemas, its the request schemas that are different.